bin/console

```diff
@@ -8,63 +8,56 @@ use Shopware\Production\HttpKernel;
 use Shopware\Production\Kernel;
 use Symfony\Bundle\FrameworkBundle\Console\Application;
 use Symfony\Component\Console\Input\ArgvInput;
-use Symfony\Component\Dotenv\Dotenv;
-use Symfony\Component\ErrorHandler\Debug;
 
-if (\PHP_VERSION_ID < 70400) {
-    echo 'Your cli is running PHP version ' . \PHP_VERSION . ' but Shopware 6 requires at least PHP 7.4.0' . \PHP_EOL;
-    exit(1);
-}
-
-set_time_limit(0);
-
-$classLoader = require __DIR__ . '/../vendor/autoload.php';
+$envFile = __DIR__ . '/../.env';
 
-if (!class_exists(Application::class)) {
-    throw new RuntimeException('You need to add "symfony/framework-bundle" as a Composer dependency.');
+if (!file_exists(__DIR__ . '/../.env')) {
+    $_SERVER['APP_RUNTIME_OPTIONS']['disable_dotenv'] = true;
```

Comment:
The runtime will automatically load the `.env` file. However, if it does not exist, it would look for a `.env.dist` file. This file is not meant to be loaded, so we prevent DotEnv if no `.env` file exists.